patterns/sso-iam-identity-center/cam/sso.tf

```diff
@@ -0,0 +1,179 @@
+# Pre requisite
+# Enable AWS IAM Identity Manager (https://console.aws.amazon.com/singlesignon/home/)
+
+provider "kubernetes" {
```

Comment:
I don't know that this belongs under `sso.tf` - perhaps we leave it in `main.tf`